test/run

```diff
@@ -137,6 +138,34 @@ function run_change_password_test() {
   fi
 }
 
+function run_change_password_new_user_test() {
+  local tmpdir=$(mktemp -d)
+  chmod -R a+rwx "${tmpdir}"
+
+  # Create MySQL container with persistent volume and set the initial password
+  create_container "testpassnewuser1" -e MYSQL_USER=user -e MYSQL_PASSWORD=foo \
+    -e MYSQL_DATABASE=db -v ${tmpdir}:/var/lib/mysql/data:Z
+  test_connection testpassnewuser1 user foo
+  docker stop $(ct_get_cid testpassnewuser1) >/dev/null
+
+  # Create second container with changed password
+  create_container "testpassnewuser2" -e MYSQL_USER=user2 -e MYSQL_PASSWORD=bar \
+    -e MYSQL_DATABASE=db -v ${tmpdir}:/var/lib/mysql/data:Z
+  test_connection testpassnewuser2 user foo
+
+  # See whether there is an error message in the log
+  if ! docker logs $(ct_get_cid testpassnewuser2) | grep 'User .*user2.* does not exist in database' ; then
```

Comment:
Why not to check 'User user2 does not exist in database` directly? I guess, wildcards are not needed.

Reply:
That's true, will fix it.

Reply:
Fixed.